depends/README.md

```diff
@@ -1,16 +1,59 @@
-### Usage
+# Depends build
+
+This is a system of building and caching dependencies necessary for building Bitcoin,
```

Comment:
s/Bitcoin/Bitcoin Core/ 

> either on your local machine or through cross-compilation

I think this needs to be rephrased, or just dropped.

Reply:
I pushed a variation.